stage0/src/sev.rs

```diff
@@ -28,14 +28,37 @@ use oak_sev_guest::{
 use spinning_top::Spinlock;
 use x86_64::{
     instructions::tlb,
-    registers::control::Cr3,
+    registers::{control::Cr3, model_specific::Msr},
     structures::paging::{
         frame::PhysFrameRange, page::AddressNotAligned, Page, PageSize, PageTable, PageTableFlags,
         PhysFrame, Size1GiB, Size2MiB, Size4KiB,
     },
     PhysAddr, VirtAddr,
 };
 
+/// The identifier for the MSR used to set the default caching mode.
+const IA32_MTRR_DEFTYPE: u32 = 0x2ff;
+
+/// Value for enanabling MTRR with write-protect mode by default.
+const MTRR_EN_WITH_WP: u64 = 0x805;
+
+/// Write a config value to IA32_MTRR_DefTpype MSR register to set the default caching mode to
+/// write-protected (WP).  This is required by Linux kernels since July, 2022, with this
+/// requirement back-ported to 5.15.X.
+///
+/// The Linux kernel gives a warning that MTRR is not setup properly, and that BOIOS proably
```

Comment:
typo: BIOS

Reply:
Also, can you copy-paste the warning one can expect?

Reply:
Done